recipes-core/easysplash/easysplash_2.0.0.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY= "Install binary and service for the easysplash"
```

Comment:
The summary ought to summarise the application goal we're packaging, not what the recipe does.